src/constants.js

```diff
@@ -0,0 +1,57 @@
+const LETTER_SCORES = {
```

Comment:
@Callumhayden99 did this very cool thing where he had the key value pairs in seperate lines ordered by value first, it's more readable imo

Reply:
I agree it's far more readable having the value as the key. It was something I considered doing but I ultimately decided just on a single key paired with a single value. Made working with the functions easier imo.